operators/Makefile

```diff
@@ -321,11 +334,12 @@ clean-e2e:
 
 ci: dep-vendor-only check-fmt generate check-local-changes unit integration e2e-compile docker-build
 
-# Run e2e tests in a dedicated gke cluster,
-# that we delete if everything went fine
+# Run e2e tests in a dedicated gke cluster, that we delete if everything went fine.
 # Let's use n1-standard-8 machine to have enough room for multiple pods on a single node.
-ci-e2e:
-	$(MAKE) bootstrap-gke dep-vendor-only e2e PSP=1 GKE_MACHINE_TYPE=n1-standard-8
+ci-e2e: ci-bootstrap-gke e2e
+
+ci-bootstrap-gke:
+	PSP=1 GKE_MACHINE_TYPE=n1-standard-8 $(MAKE) bootstrap-gke
```

Comment:
We are already using this instance type for e2e tests on CI
https://github.com/elastic/cloud-on-k8s/blob/master/build/ci/Makefile#L133

Reply:
Hm, interesting. It is ignored because we call again `$(MAKE)` again to create the gke cluster and then `GKE_MACHINE_TYPE` is not propagated. This is why we reintroduced it again [here](https://github.com/elastic/cloud-on-k8s/pull/1083/commits/79042a5fccd2a930b1116fec30ce8acf739e79e9#r294158462).

Reply:
If you will look on any cluster started from CI, then it will use this instance and it worked that way for quite a long time already. Maybe if you are trying to run e2e tests locally, then it doesn't use it, but from CI we are using `n1-standard-8` for a long time now.